### DISCUSSION TITLE
Welcome to Battle-arena Discussions!

### DISCUSSION THREAD

**Discussion by sup-ninja** [sup-ninja/Battle-arena#4] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sup-ninja** [comment]
Hello, everyone.
for enhance shoot effect, only it should change the effect image?

**Comment by mindstar0209** [comment, marked as answer]
yes, you can see my commit history.
I changed it

**Comment by sup-ninja** [comment]
Thanks.
